gms/api/src/main/pegasus/com/linkedin/ml/MLModelKey.pdl

```diff
@@ -0,0 +1,30 @@
+namespace com.linkedin.ml
+
+import com.linkedin.common.DataPlatformUrn
+import com.linkedin.common.FabricType
+
+/**
+ * Key for MLModel resource
+ */
+record MLModelKey {
+
+  /**
+   * Standardized platform urn where ML Model is defined. The data platform Urn (urn:li:platform:{dataScienceplatform_name})
```

Comment:
I am curious to know some actual examples for the model platform? Does it refer to platforms like tensorflow?

Reply:
In the same way how we have `DataPlatform` for the `Dataset` like `DataLake`, `StreamPlatform`, etc....
This could be some of the `DataPlatform` for the `MlModel` in our case we use it like `DataSciencePlatform` it could also be `TensorFlow` as well....